src/CallingExtensions.js

```diff
@@ -231,6 +232,13 @@ class CallingExtensions {
       });
     }
 
+    const isFailedEvent = String(type).endsWith("_FAILED");
+
+    if (isFailedEvent) {
+      const failedHandler = eventHandlers[messageType.FAILED];
+      failedHandler(data, event);
+    }
```

Comment:
Catch all FAILED events here

Reply:
What is the benefit of a developer using `isFailed` vs the specific handler?

Reply:
If the developer has yet to update a new failed handler in time by bumping their sdk version, they would be able to catch the failed event in this handler

Reply:
As we introduce more events to notify the apps of API failures, having to subscribe to each failed event handler does not seem sustainable in the long run